doc/man3/X509_get_subject_name.pod

```diff
@@ -74,6 +105,8 @@ earlier versions.
 X509_CRL_get_issuer() is a function in OpenSSL 1.1.0. It was previously
 added in OpenSSL 1.0.0 as a macro.
 
+X509_NAME_hash() was turned into a macro and deprecated in OpenSSL 3.0.
+
 =head1 COPYRIGHT
 
 Copyright 2015-2020 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
nit: 2021 ?

Reply:
I suppose some script will take care of required copyright changes, but done anyway.